frontend/src/db/shop_db.ts

```diff
@@ -19,42 +19,63 @@ export class AppInfoDB {
       request.onsuccess = () => resolve((this.db = request.result))
       request.onupgradeneeded = () => {
         const db = request.result
-        db.createObjectStore('appInfo', { keyPath: 'id' })
+        db.createObjectStore('appInfo', { keyPath: 'app_id' })
```

Comment:
AppInfos are now cached by app_id instead of id. (Id is the actually row-id of that app-id)

Reply:
It's the cache that stores the app infos.

Reply:
but app infos are precisely about an app_id/version combination, not just about an app.

Reply:
Do we want to display old app_infos? I thought replacing them with a newer version is intended?
To be clear: this has nothing todo with caching the .xdc files.